src/mixing_fab.cc

```diff
@@ -0,0 +1,196 @@
+#include "mixing_fab.h"
+#include <sstream>
+
+using cyclus::Material;
+using cyclus::Composition;
+using cyclus::toolkit::ResBuf;
+
+using pyne::simple_xs;
```

Comment:
You should not place these at this scope.  Usings should be within function bodies.  Also is simple_xs even used?